### DISCUSSION TITLE
How to post a comment on a ticket

### DISCUSSION THREAD

**Discussion by ghost** [jira-node/node-jira-client#351] [Q&A] [answered]
Hi, can you add an example on how to post a comment on an example Jira ticket?

**Comment by Seth10001** [comment, marked as answer]
It should just be 

```js
jira.addComment("ISSUE-xxx", "Some comment")
```

**Reply by ghost** [reply]
How can I attatch a pdf to a comment?

**Reply by Seth10001** [reply]
It [sounds like files can't be attached to comments](https://community.atlassian.com/t5/Jira-questions/How-to-attach-a-text-file-to-a-comment-to-jira-ticket-using-REST/qaq-p/1218474), however they can be attached to the issue then referenced in a comment.


So you could do something like this:
```js

jira.addAttachmentOnIssue("ISSUE-xxx",  exampleFile)

jira.addComment("ISSUE-xxx", "Here is a link to the [^example.txt] file.")
```

**Reply by ghost** [reply]
thanks Seth
